kubernetes/deployment.template.yaml

```diff
@@ -0,0 +1,178 @@
+# Kubernetes send-letter-service resource template
+#
+# All variables are replaced at deployment time with environment variables.
+#
+# The following variables are derived as follows by the pipeline. DO NOT CHANGE:
+#
+#   - NAMESPACE:    product
+#   - SERVICE_NAME: product-component-branch
+#   - IMAGE_NAME:   registry-name/hmcts/product-component:branch
+#
+---
+#
+# Send Letter Service Config
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: send-letter-config
+  namespace: ${NAMESPACE}
+data:
```

Comment:
if you use used helm instead this config would be 1 tenth the size...
This is deprecated new projects shouldn't use it

See this draft store PR for the implementation:
https://github.com/hmcts/draft-store/pull/243/files

Reply:
thanks for the link. please follow my explanation in jira ticket. in case of a strong position in using helm charts - please move back to `in progress` with a comment follow-up

Reply:
Have done so

Reply:
thank you